modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/change/tracking/spi/display/FragmentEntryLinkCTDisplayRender.java

```diff
@@ -56,7 +56,10 @@ public String getTitle(Locale locale, FragmentEntryLink fragmentEntryLink)
 
 	@Override
 	public boolean isHideable(FragmentEntryLink fragmentEntryLink) {
-		if (fragmentEntryLink.getOriginalFragmentEntryLinkId() == 0) {
+		Layout layout = _layoutLocalService.fetchLayout(
+			fragmentEntryLink.getClassPK());
```

Comment:
Use plid (page layout id) to get the layout. Currently, FragmentEntryLink is only used to create the link between fragment and pages, so the classPK happens to be the same as the plid. And unless the table only has the `classPK` column, we should always use `classPK` with `classNameId` to avoid referencing other entities with the same pk.